core/templates/dev/head/domain/feedback_thread/FeedbackThreadObjectFactory.ts

```diff
@@ -17,12 +17,28 @@
    thread domain objects.
  */
 
-var oppia = require('AppInit.ts').module;
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+export class FeedbackThread {
+  status: string;
+  subject: string;
+  summary: string;
+  originalAuthorName: string;
+  lastUpdated: number;
+  messageCount: number;
+  stateName: string;
+  threadId: string;
+  // TODO(YashJipkate): Replace 'any' with the exact type. This has been kept as
+  // 'any' because 'messages' is an array of dicts with underscore_cased keys
+  // which give tslint errors against underscore_casing in favor of camelCasing.
+  // https://github.com/oppia/oppia/issues/7176
```

Comment:
In #7176 please add a checkboxed list of files that are affected. (e.g. have one checkbox for core/templates/dev/head/domain/feedback_thread/FeedbackThreadObjectFactory.ts)

Reply:
Done!